dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -101,6 +105,9 @@ http.compression.desc=An element that is configured within the httpEndpoint elem
 http.samesite=SameSite Options
 http.samesite.desc=An element that is configured within the httpEndpoint element so that the associated HTTP channel can consider SameSite configurations.
 
+http.headers=Header Options
+http.headers.desc=An element that is configured within the httpEndpoint element so that the associated HTTP channel can consider response header configurations.
```

Comment:
Suggestion: Replace "consider" with another verb, maybe "evaluate," "assess," "examine," or "access."  Normally, "consider" is something that humans/animals do.